packages/commonwealth/client/scripts/views/modals/AuthModal/useAuthentication.tsx

```diff
@@ -439,13 +440,13 @@ const useAuthentication = (props: UseAuthenticationProps) => {
   ) => {
     try {
       if (username) {
-        await updateProfile({
-          userId: user.id,
-          // @ts-expect-error <StrictNullChecks>
-          address: account.profile.address,
-          // @ts-expect-error <StrictNullChecks>
-          chain: account.profile.chain,
+        await updateUser({
+          id: user.id.toString(),
           name: username,
+          profile: {},
+          // @mzparacha do we need to uupdate address and chain?
+          //address: account?.profile?.address,
+          //chain: account?.profile?.chain,
```

Comment:
let's make sure this will be eventually added or removed from the code

Reply:
@mzparacha can you approve this?

Reply:
yes, i think we already get address/chain by this (`onSaveProfileInfo`) point so lets update

Reply:
@mzparacha so, should we delete the comments or change the updateUser schema to add more fields that can be updated like address and chain?

Reply:
lets update the fields for which we have the data

Reply:
what I don't understand is that in the previous api (updateProfile) those fields were not updated

Reply:
I will have to check but I remember there was a time when we returned those data objects (like when we had the really old login modal). But again if we currently get that data at this stage, then its best to update userStore with it